.travis.yml

```diff
@@ -235,6 +235,17 @@ matrix:
         - USE_SYSLIBS=false
         - SHARED_LIBSCSYNTH=false
 
+    # macOS 10.13 xcode10 legacy build with qt 5.9
+    - os: osx
+      osx_image: xcode10.1
+      env:
+        - JOB_NAME="macos xcode10.1 legacy build with qt 5.9"
+        - QT=true
+        - QT_FORMULA="dyfer/qt/qt@5.9.3"
```

Comment:
where is this formula maintained? would it be easy to have an sc-org fork instead?

Reply:
I hacked this formula together, it's in [my repo](https://github.com/dyfer/homebrew-qt). Totally unofficial :) 
I'd be happy to transfer it to sc-org, but wasn't sure if we want that.